tests/test_team_api.py

```diff
@@ -125,13 +125,13 @@ def test_add_user_to_team(mocked_url, mocked_rw_apikey, mocked_account_resource_
     team_api.add_user_to_team(test_team_id, user_id_to_add)
     out, err = capsys.readouterr()
 
-    assert "Added user with id: '%s' to team\n" % user_id_to_add == out
+    assert "Added user with key: '%s' to team\n" % user_id_to_add == out
 
 @httpretty.activate
 @patch('lecli.apiutils.get_account_resource_id')
 @patch('lecli.apiutils.get_rw_apikey')
 @patch('lecli.team_api._url')
-def test_add_user_to_team(mocked_url, mocked_rw_apikey, mocked_account_resource_id, capsys):
+def test_delete_user_from_team(mocked_url, mocked_rw_apikey, mocked_account_resource_id, capsys):
```

Comment:
Deleting a user from a team is supported with PUT at the moment. DELETE for teams API supports deleting a team only.